### PR TITLE
Use i18n for download page texts

### PR DESCRIPTION
**Pull request by chingucoding**

### CODE REVIEW

File: src/i18n/locales/en.json [hunk 1]
```diff
@@ -94,5 +94,22 @@
 		"themes": "Themes",
 		"discord": "Discord",
 		"github": "Github"
+	},
+	"download": {
+		"title": "Download Files",
+		"donation_description": "Please consider donating to support our work on Files",
+		"donation_button": "Donate",
+		"microsoft_store": {
+			"title": "Microsoft Store",
```
Comment (yaira2) [inline src/i18n/locales/en.json:103]: "Microsoft Store" and "Files" should be kept in English.

Comment (chingucoding) [reply src/i18n/locales/en.json:103]: Which "Files" text do you mean?

Comment (yaira2) [reply src/i18n/locales/en.json]: Anytime it refers to the app name

Comment (chingucoding) [reply src/i18n/locales/en.json]: I think maintaining that using template strings is a lot more effort than just watching out nobody translates files in these strings. For example "Download Files" is "Files herunterladen" in german, so we would need to template string if we don't want to includes files in those strings. Or maybe I missunderstood what you mean.

Comment (yaira2) [reply src/i18n/locales/en.json]: Template strings would work

Comment (chingucoding) [reply src/i18n/locales/en.json]: Should we do that for all texts in that file or only for the download ones?

Comment (yaira2) [reply src/i18n/locales/en.json]: We should really do it for all of them

Comment (chingucoding) [reply src/i18n/locales/en.json]: There doesn't seem to be a very elegant way but it's done now.

**Comment by yaira2** [comment]
image
This doesn't look right

**Comment by chingucoding** [comment]
That should have been the only place where it was broken, fixed now.

**Review by yaira2 — APPROVED** [review]
LGTM

**Pull request merged**